mypy/typeanal.py

```diff
@@ -40,6 +40,9 @@ def analyze_type_alias(node: Expression,
     # that we don't support straight string literals as type aliases
     # (only string literals within index expressions).
     if isinstance(node, RefExpr):
+        if node.kind == UNBOUND_TVAR or node.kind == BOUND_TVAR:
+            fail_func('Invalid type "{}" for aliasing'.format(node.fullname), node)
```

Comment:
Could you make the error message clearer? I think it should explicitly state that you cannot create an alias for a type variable.